install/kubernetes/helm/subcharts/pilot/values.yaml

```diff
@@ -22,3 +22,14 @@ env:
 cpu:
   targetAverageUtilization: 80
 nodeSelector: {}
+# Provide options to allow user to configure JWT related fields.
+# Values below are in time.Duration type.
+jwtOptions:
+  # JwksURI Cache expiration time duration, individual cached JwksURI item will be removed
+  # from cache after its duration expires.
+  jwksURICacheExpiration: 24h
```

Comment:
possible drop jwks/jwt prefix, as it's already under jwt group.

Reply:
Done!